integration/mediation-tests/tests-mediator-1/src/test/resources/artifacts/ESB/server/repository/deployment/server/synapse-configs/default/api/testUrlPostFix.xml

```diff
@@ -0,0 +1,12 @@
+<api xmlns="http://ws.apache.org/ns/synapse" name="testUrlPostFix" context="/postfixapi">
```

Comment:
Shall we add the license and a new line fall all the artifacts?

Reply:
Done in https://github.com/wso2/micro-integrator/pull/1077